pyproject.toml

```diff
@@ -32,13 +32,19 @@ dependencies = [
 	'importlib_metadata >= 3.7, <5; python_version < "3.8"',
 	"mapbox_earcut >= 0.12.10",
 	"matplotlib ~= 3.2",
-	"numpy ~= 1.22",
+	"networkx ~= 3.0",
+	"numpy ~= 1.21",
 	"opencv-python ~= 4.5",
 	"pillow ~= 9.1",
 	'pygame ~= 2.1.3.dev8; python_version >= "3.11"',
 	'pygame ~= 2.0; python_version < "3.11"',
-	"scipy ~= 1.7",
+	"pyglet ~= 1.5",
+	"python-fcl >= 0.7",
+	"Rtree ~= 1.0",
+	"rv-ltl ~= 0.1.0a1",
```

Comment:
@shumbo I think this constraint will not match a future `0.2.0` release of `rv-ltl`. Should we weaken it to `rv-ltl ~= 0.1`? Or if we're happy with the current API you could make a `1.0.0` release.